app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        return "Failed to get api key"
+    city_name = "Paris"
+    url = f"http://api.weatherapi.com/v1/current.json?key={api_key}&q={city_name}"
+    response = requests.get(url)
+    if response.status_code == 200:
+        data = response.json()
+        temperature = data['current']['temp_c']
+        country_name = data["location"]["country"]
+        timing = data["location"]["localtime"]
+        condition = data["current"]["condition"]["text"]
+        print(f"{city_name}/{country_name} {timing} "
+              f"Weather {temperature} Celsius {condition}")
+        return response.json()
```

Comment:
Returning `response.json()` here is redundant if you're not using the return value. Consider removing the return statement if the function's purpose is solely to print the weather information.